hack/init_build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+. "${ROOT}/hack/utilities.sh" || { echo 'Cannot load bash utilities.'; exit 1; }
+
+VERSIONS="/tmp/.versions"
+GO_VERSION='1.7.3'
+HELM_VERSION='v2.0.0'
+GLIDE_VERSION='v0.12.3'
+
+function update-golang() {
+  # Check version of golang
+  local current_go_version="$(cat "${VERSIONS}/go" || echo "unknown")"
```

Comment:
There's certainly room to PR a fix against the helm repo (maybe `helm version --short` to indicate that you want something machine-parseable?), as this change isn't going to introduce much risk into the next release.

But, since v2.0.0 is out as @martinmaly mentioned, we'll have to wait for at least the next release for it to get in, and I'd rather not see us wait for the next release (probably [2.1.0](https://github.com/kubernetes/helm/milestone/22) on 1/7/2017, since this is a new feature) just to get this PR in.

All that being said, I think it's a good idea to "get" the helm version the hacky way for now (maybe just stick to the hard-coded vers?).

Just my $0.02